### ISSUE TITLE
Computation cost about "Codebook"

### ISSUE THREAD

**Issue by Shenyi-Z** [AdaCache-DiT/AdaCache#1]
Thanks for your excellent work!

As mentioned in the paper, AdaCache is an awesome training-free method for DiT-based video generation acceleration. I'm quite interested in the computation cost when collecting the codebook. Could you please share how much computation cost is spent on searching the optimal codebook?

**Comment by AdaCache-DiT** [comment]
Thank you for your interest in our work.

We tune our codebook hyperparameters (i.e., cache-metric thresholds, basis cache-rates) based on a small set of callibration prompts. We run the baseline video DiT, observe how the cache-metric changes across the denoising steps, and set the thresholds based on it. The basis cache-rates can be user-defined based on the required quality-latency trade-off. Each callibration run is relatively lightweight (e.g. takes around 1-2hrs on a single A100, for OpenSora 720p-2s generations with 100-steps).

**Comment by Shenyi-Z** [comment]
Thanks for your detailed explanation! This is a wonderful work!
